app/src/main/java/com/mapbox/maps/testapp/examples/LegacyOfflineActivity.kt

```diff
@@ -71,7 +73,10 @@ class LegacyOfflineActivity : AppCompatActivity() {
     )
   }
 
+  @SuppressLint("Lifecycle")
```

Comment:
All the lifecycle events of MapView have been removed, guess we don't need this anymore.

Reply:
because we kept `mapView?.onStart()` inside the `downloadComplete`